_docs/migration_guides/upgrading_to_terragrunt_0.19.x.md

```diff
@@ -118,9 +119,9 @@ terragrunt = {
 # Input variables to set for your Terraform module
 instance_type  = "t2.micro"
 instance_count = 10
-``` 
+```
```

Comment:
Any chance you could remove all the whitespace changes? Makes the PR harder to review and find the real diff.

Reply:
Yes, definitely. I made a separate commit with this change in the first place. Reverted that one